challenges/objects-arrays.js

```diff
@@ -55,11 +93,29 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+
+let i=0;
+graduates.forEach(function(){
+	contactInfo.push(graduates[i].first_name+" "+graduates[i].email);
+	i++;
+});
+
 console.log(contactInfo);
 
+console.log();
+console.log("Request 3");
+console.log();
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+let k=0;
+graduates.forEach(function(){
+	if(graduates[k].university.indexOf("Uni") >= 0){
+		uni.push(graduates[k].university);
+	}
+	k++;
+});
+
```

Comment:
This is a clever way of doing this, but the first to parameters `forEach` is the `element` and `index` of that element. So try using `forEach((element, index) => {})`